packages/components/ng-ovh-utils/src/textAccordion/directive.js

```diff
@@ -0,0 +1,14 @@
+import template from './template.html';
+
+export default /* @ngInject */ () => {
+  return {
+    restrict: 'E',
+    transclude: true,
+    controller: 'textAccordionController',
+    controllerAs: 'textAccordionCtrl',
```

Comment:
We should maintain the same design with regard to how we import / name the controllers
```javascript
import controller from './controller';

export default /* @ngInject */ () => {
  return {
    controller,
    controllerAs: '$ctrl',